Makefile

```diff
@@ -14,7 +14,7 @@ COMPONENT_CLI_IMAGE_REPOSITORY                 := $(REGISTRY)/cli
 SOURCES := $(shell go list -f '{{$$I:=.Dir}}{{range .GoFiles }}{{$$I}}/{{.}} {{end}}' ./... )
 GOPATH                                         := $(shell go env GOPATH)
 
-NOW         := $(shell date --rfc-3339=seconds | sed 's/ /T/')
+NOW         := $(shell date -u +%FT%TZ)
```

Comment:
To generate the same result it should be `date  +%FT%T%:z`

Reply:
Updated.